scripts/install

```diff
@@ -3,34 +3,52 @@
 #=================================================
 # GENERIC START
 #=================================================
-
-# Import common cmd
-source ./experimental_helper.sh
-source ./_common.sh
-
 # IMPORT GENERIC HELPERS
+#=================================================
+
+source experimental_helper.sh
+source _common.sh
 source /usr/share/yunohost/helpers
 
+#=================================================
+# MANAGE SCRIPT FAILURE
+#=================================================
+
+ynh_clean_setup () {
+	ynh_clean_check_starting
+}
 # Exit if an error occurs during the execution of the script
 ynh_abort_if_errors
 
-ynh_script_progression --message="Validating installation parameters..."
-
+#=================================================
 # RETRIEVE ARGUMENTS FROM THE MANIFEST
+#=================================================
+
 domain=$YNH_APP_ARG_DOMAIN
 path_url=$YNH_APP_ARG_PATH
 admin=$YNH_APP_ARG_ADMIN
+admin_pwd=$YNH_APP_ARG_PASSWORD
+
 db_user="pgadmin"
-admin_pwd=$YNH_APP_ARG_ADMIN_PASSWORD
+db_pwd=$(ynh_string_random --length 30)
```

Comment:
The user need this password, we can't use a random password.

Reply:
it's still here, L30